src/registry_schemas/schemas/transition.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/transition",
+  "required": [
+    "nameTranslations",
+    "offices",
+    "parties",
+    "shareStructure",
+    "hasProvisions"
+  ],
+  "type": "object",
+  "title": "Transition Filing",
+  "properties": {
+    "nameTranslations": {
+      "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_translations"
+    },
+    "offices": {
+      "registeredOffice": {
+        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+      },
+      "recordsOffice": {
+        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+      },
+      "required": ["registeredOffice", "recordsOffice"]
+    },
+    "parties": {
+      "type": "array",
+      "items": {
+        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
+      }
+    },
+    "shareStructure": {
+      "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/share_structure"
+    },
+    "hasProvisions": {
```

Comment:
Is this different than the resolutionDates or hasRightsOrRestrictions already in share structure?

Reply:
Yes, this is the pre existing company provisions that we store in the businesses table (restriction_ind column)

Reply:
right.
thanks for walking through in chat.